backend/corpora/lambdas/api/v1/common.py

```diff
@@ -35,3 +37,32 @@ def delete_dataset_common(db_session: Session, dataset: Dataset, token_info: dic
                 original.create_revision(dataset.collection.id)  # Restore the original dataset and S3 assets
             dataset.asset_deletion()  # Delete the S3 assets and database rows.
             dataset.delete()  # Delete the dataset row.
+
+
+def reshape_for_curation_api_and_is_allowed(collection, token_info, allow_access=False):
+    owner = collection["owner"]
+    if is_user_owner_or_allowed(token_info, owner):
+        collection["access_type"] = "WRITE"
+    elif not allow_access and collection["visibility"] == CollectionVisibility.PRIVATE:
+        # User neither provided the uuid for access nor are they authorized by their access token
+        return False
+    elif token_info:
```

Comment:
Remove line49-51. Default access-type is "READ". We don't need to check for the existence of token-info to set that.

Reply:
But it has to be set explicitly, right?

This is motivated by my most recent comment on the thread in [this issue](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/2083)

Reply:
we can revisit this particular issue once Brian lets us know what he wants